### ISSUE TITLE
support custom post types and custom taxonomies

### ISSUE THREAD

**Issue by asirota** [NewPath-Consulting/Wild-Apricot-Press#159]
Currently the plugin doesn't support custom post types and custom taxonomies for full access control

-on a CPT only the post part is restricted

- on a custom taxonomy we have no way to restrict access at all

**Comment by nataliebrotherton** [comment]
@asirota is this resolved?

**Comment by asirota** [comment]
I have not heard back from the client one way or another, so I am not sure. I'll reach out.

**Comment by asirota** [comment]
Actually he did say this:


Staffan Sandberg
Fri, Sep 20, 1:34 PM
to me

Hi Alex,

I just downloaded and installed version 1.0.3 and it correctly restricts access to video pages that are tagged to require an ISDE login via WildApricot.

I also tested this functionality with Category pages which, unfortunately, are either all restricted or none are restricted, but I think we expected that to be the case. I believe I have a workaround for this by creating a regular page and dropping in a Cetegory shortcode. This page can then be blocked using the WAP plugin. What doesn’t seem to work are some of the logic associated with the Category function, but I will continue experimenting. 

Thanks to you and Natalie for your work on this.

All the best,
Staffan 

Staffan Sandberg
Experience Strategies
staffan@experiencestrategies.com
